packages/reactive_ble_mobile/android/src/main/kotlin/com/signify/hue/flutterreactiveble/ble/BondingManager.kt

```diff
@@ -0,0 +1,104 @@
+package com.signify.hue.flutterreactiveble.ble
+
+import android.annotation.SuppressLint
+import android.bluetooth.BluetoothDevice
+import android.content.Context
+import android.content.Intent
+import android.content.IntentFilter
+import android.content.BroadcastReceiver
+import android.os.Build
+import android.util.Log
+import com.polidea.rxandroidble2.RxBleDevice
+import com.signify.hue.flutterreactiveble.model.BondingMode
+import io.reactivex.Completable
+import io.reactivex.disposables.Disposables
+
+/**
+ * Created by clement on 2017-08-24.
+ */
+
+class BondingFailedException : RuntimeException()
+
+object BondingManager {
```

Comment:
can you make this a class ?  I prefer  to use this so it is easier for mocking and testing than an object.  Also let's add a unites for this function

- [ ]  add unites.